thrift/src/main/thrift/atoms/chart.thrift

```diff
@@ -4,6 +4,8 @@ namespace java com.gu.contentatom.thrift.atom.chart
 
 enum ChartType {
     BAR = 0
+    LINEDISCRETE = 1
```

Comment:
do these need commas? or is that not necessary in Thrift?

Reply:
It looks like they may be optional, but I'll add them in since it looks like we have more instances where we use them than where we don't.